applications/luci-app-statistics/po/zh-cn/rrdtool.po

```diff
@@ -61,299 +62,299 @@ msgstr "响应"
 #. %di
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:10
 msgid "stat_ds_ping"
-msgstr "ping"
+msgstr "%di"
 
 #. %H: Firewall - Processed Packets
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:11
 msgid "stat_dg_title_iptables__ipt_packets"
-msgstr "iptables包"
+msgstr "%H: 防火墙 - 已处理的数据包"
 
 #. Packets/s
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:12
 msgid "stat_dg_label_iptables__ipt_packets"
-msgstr ""
+msgstr "数据包/秒"
 
 #. Chain \"%di\"
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:13
 msgid "stat_ds_ipt_packets"
-msgstr ""
+msgstr "Chain \\\"%di\\\""
 
 #. %H: Netlink - Transfer on %pi
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:14
 msgid "stat_dg_title_netlink__if_octets"
-msgstr ""
+msgstr "%H: 网络 - %pi 上的数据传输"
 
 #. Bytes/s
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:15
 msgid "stat_dg_label_netlink__if_octets"
-msgstr ""
+msgstr "字节/秒"
 
 #. Bytes (%ds)
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:16
 msgid "stat_ds_if_octets"
-msgstr ""
+msgstr "字节 (%ds)"
 
 #. %H: Netlink - Packets on %pi
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:17
 msgid "stat_dg_title_netlink__if_packets"
-msgstr ""
+msgstr "%H: 网络 - %pi 的数据包"
```

Comment:
If netlink here refers to the Linux netlink specifically, we should keep it as is.